includes/hub/class-node.php

```diff
@@ -89,4 +98,46 @@ public function get_authorization_headers( $endpoint_id ) {
 	public function decrypt_message( $message, $nonce ) {
 		return Crypto::decrypt_message( $message, $this->get_secret_key(), $nonce );
 	}
+
+	/**
+	 * Gets a collection of bookmarks for this Node
+	 *
+	 * @return array
+	 */
+	public function get_bookmarks() {
+
+		$base_url = trailingslashit( $this->get_url() );
+
+		return [
+			[
+				'label' => __( 'Dashboard', 'newspack-network' ),
+				'url'   => $base_url . 'wp-admin/',
```

Comment:
Could the node provide their `admin_url()` to the hub so this is not hardcoded?

Reply:
Well, in theory it could... When the Node is first configured, it could trigger an event that would be sent to the Hub to update this particular piece of information... seems a little too much of hassle for this.. at least at this moment